py/symbolic/proguard.py

```diff
@@ -81,7 +81,12 @@ def remap_method(self, klass: str, method: str) -> Tuple[str, str] | None:
         return output if len(output[0]) > 0 and len(output[1]) > 0 else None
 
     def remap_frame(
-        self, klass: str, method: str, line: int, parameters: str = ""
+        self,
+        klass: str,
+        method: str,
+        line: int,
+        parameters: str = "",
+        use_parameters: bool = False,
```

Comment:
you can still use `Optional` here, but then internally split that into a bool + str

Reply:
good point!! 👍🏻